app/__main__.py

```diff
@@ -0,0 +1,11 @@
+import argparse
+from .linguist import *
```

Comment:
Generic importation with `*` wouldn't be a good practice, in this case, you can just import `run` from `linguist` and `import os` separately. 

Note: Use PEP8 - Imports (https://www.python.org/dev/peps/pep-0008/#imports)